demo/breadcrumbs/index.html

```diff
@@ -0,0 +1,115 @@
+      <span>/ Breadcrumbs</span>
+    </h1>
+    <ul class="c-ctl c-ctl--header">
+      <li class="c-ctl__item">
+        <fieldset class="c-chk">
+          <input class="c-chk__input js-rtl" id="nav.ctl.rtl" type="checkbox">
+          <label class="c-chk__label c-chk__label--toggle" for="nav.ctl.rtl">RTL</label>
+        </fieldset>
+      </li>
+    </ul>
+  </header>
+  <main class="c-main c-main--page">
+    <a href="https://github.com/zendeskgarden/css-components/tree/master/packages/breadcrumbs"><img class="u-float-right u-ml-sm u-mt-sm" src="../github.svg"></a>
+    <a href="https://www.npmjs.com/package/@zendeskgarden/css-breadcrumbs"><img class="u-float-right u-mt-sm" src="https://img.shields.io/npm/v/@zendeskgarden/css-breadcrumbs.svg?style=flat-square"></a>
+    <h1 class="c-main__title">Breadcrumbs</h1>
+    <div class="c-main__body">
+      <p class="u-mb-lg">The <code class="c-code">.c-breadcrumb</code>
+        component classes are used to style breadcrumb navigation. Use
+        <a href="//zendeskgarden.github.io/css-components/bedrock/">Bedrock CSS</a>
+        to reset styling for embedded anchor tags.</p>
+      <div class="u-mb-lg">
+        <h2 class="c-main__subtitle u-fs-lg u-mb">Basic Structure</h2>
+        <p class="u-mb">The breadcrumb CSS classes correspond with the
+          following canonical structure.</p>
+        <ul class="u-list-style-disc u-mb">
+          <li>
+            <code>&lt;ol class="</code><code class="c-code">c-breadcrumb</code><code>"&gt;</code>
+            <ul class="u-list-style-disc">
+              <li>
+                <code>&lt;li class="</code><code class="c-code">c_breadcrumb__item</code><code>"&gt;</code>
+                <ul class="u-list-style-disc">
+                  <li>
+                    <code>&lt;a href="..."&gt;Link</code>
+                  </li>
+                </ul>
+              </li>
+              <li>etc.</li>
+              <li>
+                <code>&lt;li class="</code><code class="c-code">c_breadcrumb__item is-current</code><code>"&gt;Current</code>
+              </li>
+            </ul>
+          </li>
+        </ul>
+      </div>
+      <div class="u-mb-lg">
+        <h2 class="c-main__subtitle u-fs-lg u-mb">Demo</h2>
+        <p class="u-mb">Using <code class="c-code">ol</code> &gt;
+          <code class="c-code">li</code> &gt;
+          <code class="c-code">a</code> tags:</p>
+        <ol class="c-breadcrumb u-mb">
+          <li class="c-breadcrumb__item"><a href>Home</a></li>
+          <li class="c-breadcrumb__item"><a href>Resources</a></li>
+          <li class="c-breadcrumb__item"><a href>Events</a></li>
+          <li class="c-breadcrumb__item is-current">Locations</li>
+        </ol>
+        <p class="u-mb">Using <code class="c-code">nav</code> &gt;
+          <code class="c-code">a</code> tags:</p>
+        <nav class="c-breadcrumb">
+          <a class="c-breadcrumb__item" href>Home</a>
```

Comment:
This doesn't seem to be recommended and is semantically incorrect for a breadcrumb if they were independent links not related this markup is fine.

Reply:
I was perusing an [alpha version of Bootstrap](https://v4-alpha.getbootstrap.com/components/breadcrumb/) that showed this markup and just wanted to be sure the styling held up. But you're right and I'll remove it.